### ISSUE TITLE
[BUG][Editor] User is unable to distinguish between active and not active state in Editor buttons

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-react#1599]
Unable to distinguish between the active and not active state of the tools in high-contrast Aquatic/Desert themes.

Expected Result:
Users should be able to distinguish between the checked and unchecked checkboxes in high-contrast themes also. 

User Impact:
Users with low vision who rely on high contrast will not be able to distinguish between the checked and unchecked checkboxes in high-contrast Aquatic/Desert themes.

https://feedback.telerik.com/kendo-react-ui/1610829-unable-to-distinguish-between-the-checked-and-unchecked-checkboxes-in-high-contrast-aquatic-desert-themes

**Comment by PekoPPT** [comment]
As the reported behavior cannot be replicated in the latest Edge version, and we didn't receive additional details about how it can be reproduced, I am closing the issue. If someone encounters the reported behavior, please let us know and we will furhter investigate the case.
